pages/auth.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div>
+    <NuxtPage class="auth-container card" :route="route" />
+  </div>
+</template>
+
+<script setup>
+const route = useRoute()
+</script>
+
+<style lang="scss">
+.auth-container {
+  width: 24rem;
+  padding: var(--gap-xl);
+  background-color: var(--color-raised-bg);
+  border-radius: var(--radius-lg);
+  margin: 2rem auto;
+
+  h1 {
+    font-size: 1.75rem;
+    font-weight: 500;
```

Comment:
the thin font looks ugly here imo, I'd just remove this and it'll use the standard heading weight